routes/controllers/docsController.js

```diff
@@ -1,6 +1,34 @@
+const source = require('rfr');
+const MarkdownIt = require('markdown-it');
+const fs = require('fs');
+
+const { setupCommands } = source('bot/scanner/commandHandler');
+const md = new MarkdownIt();
+
+const gettingStartedMD = fs.readFileSync('docs/GettingStarted.md', 'utf-8');
+const configurationsMD = fs.readFileSync('docs/Configurations.md', 'utf-8');
+const roleReactionsMD = fs.readFileSync('docs/RoleReactions.md', 'utf-8');
+const spotifyPluginMD = fs.readFileSync('docs/SpotifyPlugin.md', 'utf-8');
+const gettingHelpMD = fs.readFileSync('docs/GettingHelp.md', 'utf-8');
```

Comment:
Could probably render the text here instead of whenever the page is loaded.

Reply:
Done